<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WIN-TH3CRUOPQ3G" ignored="0" name="ApiAutoDemo" tests="5" failures="1" timestamp="15 五月 2021 07:43:41 GMT" time="6.627" errors="0">
  <testcase name="Test01" time="0.001" classname="com.qa.tests.GetTest01"/>
  <testcase name="Test02" time="0.0" classname="com.qa.tests.GetTest01"/>
  <testcase name="Test03" time="0.001" classname="com.qa.tests.GetTest01"/>
  <testcase name="Test04" time="1.042" classname="com.qa.tests.GetTest01">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method com.qa.tests.GetTest01.Test04() didn&amp;apos;t finish within the time-out 1000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method com.qa.tests.GetTest01.Test04() didn't finish within the time-out 1000
at java.lang.Thread.interrupt(Thread.java:916)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 2 stack frames]]>
    </failure>
  </testcase> <!-- Test04 -->
  <testcase name="getListUsers" time="5.322" classname="com.qa.tests.GetTest01"/>
</testsuite> <!-- ApiAutoDemo -->
